e2e/pages/adf/process_services/widgets/checkbox.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FormFields = require('../formFields');
+import Util = require('../../../../util/util');
+import { by, element } from 'protractor';
+
+export class Checkbox {
```

Comment:
Some of the widgets classes contain Widget in the name, like TextWidget, and others don't. I suggest all to be the same, either contain it, either don't.